drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/editor/RuleModellerActionSelectorPopup.java

```diff
@@ -208,8 +209,8 @@ public void execute() {
     }
 
     //Add update, not modify
-    private void addUpdateNotModify() {
-        List<String> vars = model.getAllLHSVariables();
```

Comment:
@manstis I got back to this PR because of some fails in our selenium tests, I think this change was not needed. Could you please confirm? I will create a PR then.